solutions/project/spoke-project-unclass/hostvpcproject.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: compute.cnrm.cloud.google.com/v1beta1
+kind: ComputeSharedVPCHostProject
+metadata:
+  name: project-id-host-vpc # kpt-set: ${project-id}-host-vpc
+  namespace: projects
+  annotations:
+    cnrm.cloud.google.com/blueprint: 'kpt-fn'
```

Comment:
remove this annotations in every file : cnrm.cloud.google.com/blueprint: 'kpt-fn'

Reply:
done